src/credentials/attestation_verifier/TestDACRevocationDelegateImpl.cpp

```diff
@@ -30,13 +31,50 @@ namespace chip {
 namespace Credentials {
 
 namespace {
+
+static constexpr uint32_t kMaxIssuerBase64Len = BASE64_ENCODED_LEN(kMaxCertificateDistinguishedNameLength);
+
 CHIP_ERROR BytesToHexStr(const ByteSpan & bytes, MutableCharSpan & outHexStr)
 {
     Encoding::HexFlags flags = Encoding::HexFlags::kUppercase;
     ReturnErrorOnFailure(BytesToHex(bytes.data(), bytes.size(), outHexStr.data(), outHexStr.size(), flags));
     outHexStr.reduce_size(2 * bytes.size());
     return CHIP_NO_ERROR;
 }
+
+CHIP_ERROR X509_PemToDer(const std::string & pemCert, MutableByteSpan & derCert)
```

Comment:
Do we have to handle PEM at all here? Usually we let users do the conversion via chip-cert or openssl and just have the SDK code handle DER.

Reply:
CRL signer certificate or CRL signer delegator certificate are stored as PEM in the DCL, [generate-revocation-set.py](https://github.com/project-chip/connectedhomeip/blob/master/credentials/generate-revocation-set.py) generates the revocation set and stores them into json file, SDK reads that json file, so for now, we are stick with that choice.

We can extract the middle part from the cert in script and store it, so only need to base64 decode to make it DER.

Reply:
It could be converted to DER in the JSON files for the test version. Let's keep that as a follow-up though.

Reply:
Raised #36625 for tracking